### ISSUE TITLE
Associated Model's 'get' method and 0 val

### ISSUE THREAD

**Issue by chiplay** [dhruvaray/backbone-associations#137]
Hey @dhruvaray!

Hope all is well.  Quick question for you - the 0.6.1 [get() fn](https://github.com/dhruvaray/backbone-associations/blob/v0.6.1/backbone-associations.js#L170) for the associated model looks like this:

```
        // Get the value of an attribute.
        get:function (attr) {
            var obj = ModelProto.get.call(this, attr);
            return obj ? obj : this._getAttr.apply(this, arguments);
        },
```

I'm running into an issue where our mutated property is returning a '0' value and therefor 'obj' above is falsy and the this._getAttr circumvents bb.mutators and retrieves a cached value directly from attributes. 

I see that the logic has changed in master - but I'm still not sure it would work for our situation.  Any thoughts on what the above code doesn't more explicitly check for obj being null/undefined?

Thanks!

**Comment by dhruvaray** [comment]
Hey @chiplay! Thank you. I am doing well. You are right. We should check explicitly for being null/undefined. Do you want to submit a PR? I will be happy to merge. The changes on the trunk are for some additional bits of functionality and are orthogonal to what you have raised.

**Comment by dhruvaray** [comment]
Fixed via 8b4a41f378a5f753191f401eea25740f1979803f & 33d4d554e13b4a3c2bb9a5c8dae65c0635403299

Let me know if this works for you.

**Comment by dhruvaray** [comment]
This fix is part of 0.6.2. Closing...Please reopen if it does not yet work for you
